internal/services/compute/virtual_machine_implicit_data_disk_from_source_test.go

```diff
@@ -0,0 +1,992 @@
+package compute_test
```

Comment:
And this filename wants to be `virtual_machine_implicit_data_disk_from_source_resource_test.go`

Reply:
updated. Thanks.